public/electron.js

```diff
@@ -82,15 +83,31 @@ function initialize(queues) {
   )
 
   mainWindow.webContents.on('did-finish-load', () => mainWindow.show())
-  // mainWindow.webContents.openDevTools()
+
+  // open devtools if DEV_TOOLS is set to 1
+  if (parseInt(process.env.DEV_TOOLS)) {
+    mainWindow.webContents.openDevTools()
+  }
+
   mainWindow.on('closed', () => {
     mainWindow = null
   })
 }
 
 app.on('ready', async () => {
 
-  globalShortcut.register('CommandOrControl+Q', () => app.quit())
+  globalShortcut.register('CommandOrControl+Q', () => {
+
+    // prompt user before closing the application if there are active jobs
+    if (Object.keys(queues['queues']['video transcoding']['childPool'].retained).length != 0) {
+      const dialogOptions = {type: 'info', buttons: ['Cancel', 'Quit'], message: 'There are active jobs running, quit anyway?'}
```

Comment:
No need for `dialogOptions` to be a variable if it's only referenced once. You can just pass the object directly. Let's also break this over multiple lines for readability.

Reply:
Would the type `'question'` be more suitable here?

Reply:
Let's change the message to something a bit more descriptive perhaps:

```
'Looks like publish50 is currently running some tasks. Quitting publish50 will abort these tasks. Are you sure you want to quit?'
```

Reply:
Could you also post a screenshot of the dialog box please?

Reply:
<img width="1595" alt="Screen Shot 2020-01-22 at 1 40 03 PM" src="https://user-images.githubusercontent.com/10591665/72923278-bb5c7a00-3d1c-11ea-8042-8acc5e15956b.png">

Reply:
> Would the type `'question'` be more suitable here?

Good catch, and I think the type 'warning' suits better.